### DISCUSSION TITLE
Support for 1Password Secrets API?

### DISCUSSION THREAD

**Discussion by dmonder** [gerardog/gsudo#142] [Ideas]
I would love to see this be able to use the 1Password Secrets API to access the credential information. This would allow me to use my already existing password repository for elevation. Maybe it could be driven by an environment variable or setting something in the configuration?

**Comment by gerardog** [comment]
Hi @dmonder.
Unix/Linux sudo solves this by calling an external app/script specified in the --askpass parameter.
https://www.sudo.ws/docs/man/sudo.man/#A

Have you used the API before? Do you have a script o app that, when called, outputs your password?
